en/02_Development_environment.adoc

```diff
@@ -57,8 +60,13 @@ image::/images/glfw_directory.png[]
 Unlike DirectX 12, Vulkan does not include a library for linear algebra operations, so we'll have to download one.
 http://glm.g-truc.net/[GLM] is a nice library that is designed for use with graphics APIs and is also commonly used with OpenGL.
 
-GLM is a header-only library, so just download the https://github.com/g-truc/glm/releases[latest version] and store it in a convenient location.
-You should have a directory structure similar to the following now:
+GLM is a header-only library, so just download the https://github.com/g-truc/glm/releases[latest version] and store it in a convenient location like `C:\Vulkan-Tutorial`.
```

Comment:
Can even mention using submodules.

Reply:
Yeah, I didn't want to change too much aside from the most pressing issues. I think once we have a proper CMake setup using FetchContent we can make this whole setup chapter a lot easier :)